File: 2235-capitalize-the-title/capitalize-the-title.go
import "strings"

func capitalizeTitle(title string) string {
    words := strings.Fields(strings.ToLower(title))

    for i, word := range words {
        if len(word) >= 3 {
            runes := []rune(word)
            runes[0] = []rune(strings.ToUpper(string(runes[0])))[0]
            words[i] = string(runes)
        } 
    }
    return strings.Join(words, " ")
}

File: 2235-capitalize-the-title/capitalize-the-title.cpp
class Solution {
public:
    string capitalizeTitle(string title) {
        int n = title.size();
        for(char &c : title) {
            c = tolower(c);
        }
        for(int i=0;i<title.size(); i++) {
            int first = i;
            while(i<n && title[i] != ' ') {
                title[i] = tolower(title[i]);
                i++;
            }
            if(i - first > 2) {
                title[first] = toupper(title[first]);
            }
        }
        return title;
    }
};